src/index.ts

```diff
@@ -6,8 +6,14 @@ const logger = require('./services/logger')
 export { PocketGatewayApplication }
 
 export async function main(options: ApplicationConfig = {}): Promise<PocketGatewayApplication> {
+  options.env = { load: true }
```

Comment:
Can't we just assign this by default on the method definition and set it as a constant at the top of the file too? something like
```js
const DEFAULT_APPLICATION_OPTIONS = { load: true }
```